js/jquery.fileupload-angular.js

```diff
@@ -24,6 +24,17 @@
             './jquery.fileupload-video',
             './jquery.fileupload-validate'
         ], factory);
+    } else if (typeof exports === 'object') {
+        // Node/CommonJS:
+        factory(
+            require('jquery'),
+            require('angular'),
+            require('./jquery.fileupload'),
```

Comment:
This line is not necessary, as e.g. the validate dependency itself already requires `'./jquery.fileupload-process'`, which in turn requires `'./jquery.fileupload'`.

`jquery.fileupload-validate.js` does however miss the `'./jquery.fileupload-process'` dependency in the Node-style imports.